src/bt_auto_dumper/_v2/__main__.py

```diff
@@ -45,20 +56,43 @@ def dump_and_upload(walletname, wallethotkey, subnet_identifier: str, autovalida
                 f.write(result.stdout)
             output_files.append(output_file)
 
-    zip_filename = "output.zip"
+    zip_filename = f"{normalized_subnet_identifier}-output.zip"
     with zipfile.ZipFile(zip_filename, "w") as zipf:
         for file in output_files:
             zipf.write(file)
-    send_to_autovalidator(zip_filename, wallet, autovalidator_address, note, subnet_identifier)
-
-
-def make_signed_request(method, url, headers, files, wallet):
+    send_to_autovalidator(zip_filename, wallet, autovalidator_address, note, normalized_subnet_identifier)
+
+
+def make_signed_request(method: str, url: str, headers: dict, file_path: str, wallet: bt.wallet) -> requests.Response:
+    """
+    Make a signed request to the AutoValidator
+    Args:
+        method: HTTP method
+        url: URL
+        headers: HTTP headers
+        file_path: File path
+        wallet: Wallet object
+    Returns:
+        Response object
+    Example:
+        make_signed_request(
+            "POST",
+            "http://localhost:8000/api/v1/files/",
+            {"Note": "Test"},
+            {"file": open("test.zip", "rb")},
+            wallet
+        )
+    """
     headers["Nonce"] = str(time.time())
     headers["Hotkey"] = wallet.hotkey.ss58_address
-    file_content = files.get("file").read()
-    files.get("file").seek(0)
+    files = {"file": open(file_path, "rb")}
+    file = files.get("file")
+    file_content = b""
+    if isinstance(file, BufferedReader):
+        file_content = file.read()
+        file.seek(0)
```

Comment:
```suggestion
    file_content = file.read()
    file.seek(0)
```
this will never not happen